### PR TITLE
100.6 Sample: Edit KML ground overlay

### PR DESCRIPTION
**Pull request by ZackAllen**

### CODE REVIEW

File: src/Android/Xamarin.Android/Samples/Layers/EditKmlGroundOverlay/readme.metadata.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        "KML",
+        "KMZ",
+        "Keyhole",
+        "OGC",
+        "imagery"
+    ],
+    "nuget_packages": {
+        "Esri.ArcGISRuntime": "100.6.0"
+    },
+    "offline_data": [],
```
Comment (nCastle1) [inline src/Android/Xamarin.Android/Samples/Layers/EditKmlGroundOverlay/readme.metadata.json]: It looks like this is missing the offline data used by the sample

File: src/Forms/Shared/Samples/Layers/EditKmlGroundOverlay/EditKmlGroundOverlay.xaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    x:Class="ArcGISRuntimeXamarin.Samples.EditKmlGroundOverlay.EditKmlGroundOverlay"
+    xmlns="http://xamarin.com/schemas/2014/forms"
+    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+    xmlns:esriUI="clr-namespace:Esri.ArcGISRuntime.Xamarin.Forms;assembly=Esri.ArcGISRuntime.Xamarin.Forms">
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+        <esriUI:SceneView x:Name="MySceneView" />
```
Comment (nCastle1) [inline src/Forms/Shared/Samples/Layers/EditKmlGroundOverlay/EditKmlGroundOverlay.xaml]: I recommend adding an explicit `Grid.Row="0"` here.

Additionally, I recommend explicitly setting a height for the row definition above.

File: src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/EditKmlGroundOverlay/EditKmlGroundOverlay.xaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+﻿<UserControl
+    x:Class="ArcGISRuntime.WPF.Samples.EditKmlGroundOverlay.EditKmlGroundOverlay"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:esri="http://schemas.esri.com/arcgis/runtime/2013">
+    <Grid>
+        <esri:SceneView x:Name="MySceneView" />
+        <Border Style="{StaticResource BorderStyle}">
+            <Grid>
```
Comment (nCastle1) [inline src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Layers/EditKmlGroundOverlay/EditKmlGroundOverlay.xaml]: I'd consider putting the slider and label on the same line. I'd also consider showing the current value (basically copying what's done in the map rotation sample)

**Comment by ZackAllen** [comment]
Added the value display for all platforms.

**Pull request merged**